db/data/20200115145845_backfill_geckoboard_data.rb

```diff
@@ -0,0 +1,13 @@
+class BackfillGeckoboardData < ActiveRecord::Migration[6.0]
+  def up
+    submitted_claim_ids = Claim.submitted.pluck(:id)
+    paid_claim_ids = Claim.where.not(payment: nil).pluck(:id)
+
+    RecordClaimEventJob.perform_later(submitted_claim_ids, :submitted, :submitted_at)
+    RecordClaimEventJob.perform_later(paid_claim_ids, :paid, :paid_at)
```

Comment:
Should this now be this?

```suggestion
    RecordClaimEventJob.perform_later(paid_claim_ids, :paid, : scheduled_payment_date)
```

Reckon it's worth re-running this data migration _and_ having the jobs run to verify it all actually marry's up.

Reply:
Well spotted 👍 All seems to be in order when running it